src/app/service/poap/StartPOAP.ts

```diff
@@ -350,5 +354,5 @@ const startTwitterPOAPFlow = async (ctx: CommandContext, guildMember: GuildMembe
 	if (result.ok != 1) {
 		await guildMember.send('One more thing, there was a problem setting counting you as a participants.. You just might miss the POAP, sorry.');
```

Comment:
'... there's a problem. Could be you. Could be me. But looks like you weren't setup properly to receive a POAP. Reach out the the event coordinator for your badge.'